ports/deltarune/Deltarune.sh

```diff
@@ -72,7 +72,7 @@ check_patch() {
 check_patch
 
 # Assign gptokeyb and load the game
-$GPTOKEYB "gmloadernext.aarch64" xbox360 &
+$GPTOKEYB "gmloadernext.aarch64" -k &
```

Comment:
You can just remove the `-k` flag all together, it is superfluous as discussed here: https://discord.com/channels/1122861252088172575/1300971977288192012/1384873363201331241

```suggestion
$GPTOKEYB "gmloadernext.aarch64" &
```